server/websocket.go

```diff
@@ -987,6 +987,24 @@ func (p *Plugin) WebSocketMessageHasBeenPosted(connID, userID string, req *model
 			return
 		}
 		msg.Data = []byte(msgData)
+	case clientMessageTypeCaption:
```

Comment:
Unless there's a good reason to do it here, it would be best to handle the message in `handleClientMsg` like for most other client initiated events.

Reply:
I would say the good reason is: it avoids an unnecessary data->json string->data conversion, which isn't needed. I realize we do that from client to backend, but I can't think of a reason to do it from one backend to another. Plus, it makes sense to treat this differently than a typical message from a regular client, since it's not really a regular client.

Reply:
Fair enough. We should still call `p.metrics.IncWebSocketEvent` as appropriate though.